javacutil/src/main/java/org/checkerframework/javacutil/TreeUtils.java

```diff
@@ -2633,8 +2647,27 @@ public static ExpressionTree yieldTreeGetValue(Tree yieldTree) {
      * @return true if the variableTree is declared using the {@code var} Java keyword
      */
     public static boolean isVariableTreeDeclaredUsingVar(VariableTree variableTree) {
+        // The JCVariableDecl class has the method "declaredUsingVar()" returns true if the
+        // variable was declared using the "var" keyword.
+        // The start positions are set for types of variables declared using var in
+        // https://github.com/openjdk/jdk/commit/e2f736658fbd03d2dc2186dbd9ba9b13b1f1a8ac
+        // Checking the pos is equal to Position.NOPOS will return the wrong result after above
+        // Javac change.
         JCExpression type = (JCExpression) variableTree.getType();
-        return type != null && type.pos == Position.NOPOS;
+        if (type == null) {
+            return false;
+        } else if (JCVARDECL_DECLAREDUSINGVAR != null && !isErroneousTree(type)) {
+            try {
+                Object result = JCVARDECL_DECLAREDUSINGVAR.invoke(variableTree);
+                return Boolean.TRUE.equals(result);
+            } catch (InvocationTargetException | IllegalAccessException e) {
+                throw new BugInCF(
+                        "TreeUtils.isVariableTreeDeclaredUsingVar: reflection failed for tree: %s",
```

Comment:
```suggestion
                        e, "TreeUtils.isVariableTreeDeclaredUsingVar: reflection failed for tree: %s",
```

Including the original exception should make the error more helpful.